saleor/payment/gateways/stripe/utils.py

```diff
@@ -50,3 +55,19 @@ def get_currency_from_stripe(currency):
     Stripe's currency is using lowercase while Saleor is using uppercase.
     """
     return currency.upper()
+
+
+def get_payment_billing_fullname(payment: Payment):
+    # Get billing name from payment
+    return '%s %s' % (
+        payment.billing_last_name, payment.billing_first_name)
+
+
+def shipping_address_to_dict(shipping_address: Address):
```

Comment:
I know I didn't test for this though it is covered by tests. As now it is refactored and moved to separate utils, it will be nice to add tests for these two utils ```shipping_address_to_dict``` and ```get_payment_billing_fullname```, which will make sure the adddress are properly translated from saleor's address format to Stripe's address format, such as ```country_area -> state``` or ```street_address_1 -> line1```.

Reply:
Definitively.